src/lib/viewers/controls/slider/VolumeSliderControl.tsx

```diff
@@ -0,0 +1,167 @@
+
+        // Calculate distance from bottom of slider
+        const sliderBottom = sliderTop + sliderHeight;
+        const distanceFromBottom = sliderBottom - clientY;
+
+        return Math.max(5, distanceFromBottom);
+    }, []);
+
+    const getPositionValue = React.useCallback(
+        (pageY: number, clientY: number) => {
+            const { current: sliderEl } = sliderElRef;
+            if (!sliderEl) return 0;
+            const { height: sliderHeight } = sliderEl.getBoundingClientRect();
+            const positionRelativeToSlider = getPositionRelativeToSlider(clientY);
+            const newValue = (positionRelativeToSlider / sliderHeight) * max;
+            return Math.max(min, Math.min(newValue, max));
+        },
+        [getPositionRelativeToSlider, max, min],
+    );
+
+    const handleKeydown = (event: React.KeyboardEvent): void => {
+        const key = decodeKeydown(event);
+        if (key === 'ArrowDown') {
+            event.stopPropagation(); // Prevents global key handling
+            onUpdate(Math.max(min, Math.min(value - step, max)));
+        }
+
+        if (key === 'ArrowUp') {
+            event.stopPropagation(); // Prevents global key handling
+            onUpdate(Math.max(min, Math.min(value + step, max)));
+        }
+    };
+
+    const handleMouseDown = (event: React.MouseEvent<Ref>): void => {
+        const { button, ctrlKey, metaKey, pageY, clientY, stopPropagation } = event;
+        if (button > 1 || ctrlKey || metaKey) return;
+        onUpdate(getPositionValue(pageY, clientY));
+        setIsScrubbing(true);
+        // Prevent clicking on the slider from triggering the mouse down event on the parent slider track
+        event.stopPropagation();
+    };
+
+    const handleMouseMove = ({ pageY, clientY }: React.MouseEvent<Ref>): void => {
+        const { current: sliderEl } = sliderElRef;
+        const { height: sliderHeight } = sliderEl ? sliderEl.getBoundingClientRect() : { height: 0 };
+        onMove(getPositionValue(pageY, clientY), getPositionRelativeToSlider(clientY), sliderHeight);
+    };
+
+    const handleTouchStart = ({ touches }: React.TouchEvent<Ref>): void => {
+        onUpdate(getPositionValue(touches[0].pageY, touches[0].clientY));
+        setIsScrubbing(true);
+    };
+
+    React.useEffect(() => {
+        const handleDocumentMoveStop = (): void => setIsScrubbing(false);
+        const handleDocumentMouseMove = (event: MouseEvent): void => {
+            if (!isScrubbing || event.button > 1 || event.ctrlKey || event.metaKey) return;
+
+            event.preventDefault();
+            onUpdate(getPositionRelativeToSlider(event.clientY));
+        };
+        const handleDocumentTouchMove = (event: TouchEvent): void => {
+            if (!isScrubbing || !event.touches || !event.touches[0]) return;
+
+            event.preventDefault();
+            onUpdate(getPositionRelativeToSlider(event.touches[0].clientY));
+        };
+
+        if (isScrubbing) {
```

Comment:
Should we be removing these if `isScrubbing` is false?

Reply:
Like the other ones. Didn't want to change this. Removing this actually broke it.